database/database.go

```diff
@@ -77,3 +77,46 @@ func InsertUserProfile(user *model.UserProfile) error {
 	}
 	return err
 }
+
+//GenerateToken func to add token in db
+func GenerateToken(username string) error {

```

Comment:
Create two separate functions GenerateToken(username string) string and InsertUserAuth(userAuth *UserAuth) error